### PR TITLE
Add unauth prescription refills content

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/10264

This PR adds unauth prescription refills content as a React widget.

## Testing done
Adds unit tests for React widget.

## Screenshots

localhost_3001_dummy-page_ (1)

## Acceptance criteria
- [x] Add unauth prescription refills content as a React widget.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+    <h1>VA Prescription Refill and Tracking</h1>
+    <div className="va-introtext">
+      <p>
+        With our VA Prescription Refill and Tracking tool, you can refill your
+        VA prescriptions, track their delivery, and create lists to organize
+        your medicines. Find out if you’re eligible and how to sign up to begin
+        using this tool in our health management portal within My HealtheVet.
+      </p>
+    </div>
+    <div
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js:16]: We should probably just drop the CTA widget in here

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+        <div
+          data-entity-id={3270}
+          itemProp="acceptedAnswer"
+          itemScope
+          itemType="http://schema.org/Answer"
+        >
+          <div itemProp="text">
+            <div
+              data-template="paragraphs/wysiwyg"
+              data-entity-id={3271}
```
Comment (ncksllvn) [inline src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js:330]: I think you can sanitize this a bit by removing the `data-` attributes since it's React-based now

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      'How long will my prescriptions take to arrive, and when should I',
+    );
+    expect(text).to.include(
+      'Will my personal health information be protected?',
+    );
+    expect(text).to.include('What if I have more questions?');
+
+    wrapper.unmount();
+  });
+});
```
Comment (ncksllvn) [inline src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js:32]: You could abstract this content into an `Unauth` component and then just test for that component from this test. I'm actually curious if Mocha allows this since there is a  SCSS import in App

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+export default (store, widgetType) => {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+  if (root) {
+    import(/* webpackChunkName: "App" */
+    './components/App').then(module => {
+      const App = module.default;
+      ReactDOM.render(
+        <Provider store={store}>
+          <App />
+        </Provider>,
```
Comment (ncksllvn) [inline src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/index.js:15]: We talked about this over Slack but I'm still curious if `<App store={store}/>` works the same way as wrapping in a `Provider`

File: src/applications/static-pages/widgetTypes.js [hunk 5]
```diff
@@ -15,4 +15,5 @@ export default {
   VIEW_DEPENDENTS_CTA: 'view-dependents-CTA',
   FORM_686_CONTENT_REVEAL: '686UnauthContentBlock',
   CAREGIVER_CONTENT_TOGGLE: 'caregiver-content-toggle',
+  PRESCRIPTIONS_REFILL_CONTENT: 'prescriptions-refill-content',
```
Comment (ncksllvn) [inline src/applications/static-pages/widgetTypes.js:18]: This will need to be registered in vets-api as well to appear as a Flipper option

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: src/applications/static-pages/widgetTypes.js (hunk 5 above)
Comment (kelsonic) [reply src/applications/static-pages/widgetTypes.js:18]: Oh awesome! Do you have a code snippet/PR/etc. you share how that's been done elsewhere for another widget?

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js (hunk 2 above)
Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js:330]: Good idea! I will do this for all the PRs

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js (hunk 3 above)
Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js:32]: I'm not really sure what you mean by an `Unauth` component 🙂 Could you rephrase?

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/index.js (hunk 4 above)
Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/index.js:15]: I think that would work but only if the root element of `App` were a `Provider`, right? I can move that Provider logic into App if you think that'd be cleaner! 🙂

File: src/applications/static-pages/widgetTypes.js (hunk 5 above)
Comment (ncksllvn) [reply src/applications/static-pages/widgetTypes.js:18]: it's this file, https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/features.yml

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/index.js (hunk 4 above)
Comment (ncksllvn) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/index.js:15]: Nah it's cool the way it is. For some reason other widgets are able to get away without using a Provider 🤷

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js (hunk 3 above)
Comment (ncksllvn) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js:32]: Yeah I just mean chunking out all of the content out of App and into a single `UnauthenticatedContent` view so that -

```
if (featureEnabled === false) return <UnauthenticatedContent/>
```

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js [hunk 6]
```diff
@@ -0,0 +1,415 @@
...
+                  Find out how to apply for VA health care
+                </a>
+              </p>
+              <p>
+                <strong>And you must have one of these free accounts:</strong>
+              </p>
+              <ul>
+                <li>
+                  A{' '}
+                  <a href="">
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js:118]: Missing link

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js (hunk 2 above)
Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js]: Done! 🎉

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js (hunk 3 above)
Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.unit.spec.js:32]: So I would do that if the unauthenticated content differed considerably from the authenticated content, but i would say around 80% of it is the same currently, so I'll defer this request until the content begins to diverge more if that's ok! 🙂

File: src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js (hunk 6 above)
Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js:94]: @DanielleThierryUSDSVA Any idea where this link should go? 🙂

Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js:94]: https://www.myhealth.va.gov/mhv-portal-web/upgrading-your-my-healthevet-account-through-in-person-or-online-authentication

Comment (kelsonic) [reply src/applications/static-pages/health-care-manage-benefits/prescriptions-refill-content/components/App/index.js:94]: Do not open in new tab (e.g. no `target="_blank"`) 🙂
